packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -0,0 +1,42 @@
+import { ApiClient } from '../.'
+import BigNumber from 'bignumber.js'
+
+/**
+ * Loan RPCs for DeFi Blockchain
+ */
+export class Loan {
+  private readonly client: ApiClient
+
+  constructor (client: ApiClient) {
+    this.client = client
+  }
+
+  /**
+   * Updates an existing loan scheme.
+   *
+   * @param {number} minColRatio Minimum collateralization ratio
+   * @param {BigNumber} interestRate Interest rate
+   * @param {UpdateLoanSchemeOptions} options
+   * @param {string} options.id Unique identifier of the loan scheme, max 8 chars
+   * @param {number} [options.activateAfterBlock] Block height at which new changes take effect
+   * @param {UTXO[]} [options.utxos = []] Specific UTXOs to spend
+   * @param {string} options.utxos.txid Transaction Id
+   * @param {number} options.utxos.vout Output number
+   * @return {Promise<string>} Hex string of the transaction
+   */
+  async updateLoanScheme (minColRatio: number, interestRate: BigNumber, options: UpdateLoanSchemeOptions): Promise<string> {
```

Comment:
Since the `id` is the unique identifier to identify the updating loan scheme, maybe take the `id` out of `UpdateLoanSchemeOptions` and put another parameter to it.
seems a bit odd when calling the function here ->

Reply:
It will look like this
`async updateLoanScheme (scheme: UpdateLoanScheme, utxos: UTXO[] = []): Promise<string>`